src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -429,16 +540,10 @@ def executeSingleMCPIteration(self, test_type, iteration_content, sas_test_harne
     # Step 8 : Trigger CPAS and wait until completion
     self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
 
-    # Step 9 : Invoke IAP reference model
```

Comment:
Please add a note saying that this will happen as part of step 12, which is OK.

Reply:
Added the comment